luigi/contrib/esindex.py

```diff
@@ -169,15 +169,18 @@ def touch(self):
 
         The document id would be sufficient but,
         for documentation,
-        we index the parameters `update_id`, `target_index`, `target_doc_type` and `date` as well.
+        we index the parameters `update_id`, `target_index`, `target_doc_type` and `date` as well.  `date_utc` added
```

Comment:
I think the added comment fits better as a PR comment instead of docstring to track the reasoning. Maybe something as simple as: we index the parameters `update_id`, `target_index`, `target_doc_type` , `date` and `date_utc`.